index.html

```diff
@@ -1,31 +1,201 @@
-    <nav class="hamburger"><i class="fas fa-bars"></i></nav>
-   </header>
-
-   <section class="headline">
-    <h1 class="intro">Hey There. I'm Daniel</h1>
-   <h2 class="developer">I'm a Software developer</h2>
-   <p class="abilities">I can help you build a product , feature or website Look through some of my work and experience! 
-    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    <ul class="icons">
-        <li class="github"><a href=""><i class="fa-brands fa-github"></i></a></li>
-        <li class="linkedin"><a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-        <li class=angelist><a href=""><i class="fab fa-angellist"></i></a></li>
-        <li class="twitter"><a href=""><i class="fa-brands fa-twitter"></i></a></i></li>
-        <li class="facebook"><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
-    </ul>
-   </section>
+    <header class="header">
+        <a href="" class="name">Daniel</a>
+        <nav class="hamburger"><i class="fas fa-bars"></i></nav>
+    </header>
+
+    <section class="headline">
+        <h1 class="intro">Hey There. I'm Daniel</h1>
+        <h2 class="developer">I'm a Software developer</h2>
+        <p class="abilities">I can help you build a product , feature or website Look through some of my work and
+            experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <ul class="icons">
+            <li class="github"><a href=""><i class="fa-brands fa-github"></i></a></li>
+            <li class="linkedin"><a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+            <li class=angelist><a href=""><i class="fab fa-angellist"></i></a></li>
+            <li class="twitter"><a href=""><i class="fa-brands fa-twitter"></i></a></i></li>
+            <li class="facebook"><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+        </ul>
+    </section>
+
+    <section class="recent">
+        <h2 class="recent-item">My Recent Works</h2>
+        <div class="line"></div>
+        <div class="yellowboard">
+            <img src="images/bigyellow.png" width=327 height=237 alt="">
+        </div>
+        <div class="topic">
+            <div class="stories">Multi-Post Stories</div>
+            <p class="stories-text">A daily selection of privately personalized reads;
+                no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="skills">
+                <button><li class="js" id="btn">css</li></button>
+                <button><li class="js" id="btn">html</li></button>
+                <button><li class="js" id="btn">javascript</li></button>
+            </ul>
+        </div>
+       <a href=""><button type="button" class="btnsettings">See project</button></a> 
```

Comment:
- [ ] [OPTIONAL] Good work using the `<button>/<a>` tag on this link/button however you should use only one so consider showing between  `<button>` or `<a>`